libs/sdk-react-native/src/index.tsx

```diff
@@ -92,6 +97,18 @@ export type BitcoinAddressData = {
     message?: string
 }
 
+export type Config = {
+    breezserver: string
+    mempoolspaceUrl: string
+    workingDir: string
+    network: Network
+    paymentTimeoutSec: number
+    defaultLspId?: string
+    apiKey?: string
+    maxfeeSat?: number
+    maxfeepercent: number
```

Comment:
nit: shouldn't this be in camel case `maxfeePercent`?

Reply:
This is a direct copy of the naming from the SDK

Reply:
Good point, then all good